Orange/data/table.py

```diff
@@ -1133,6 +1133,99 @@ def _filter_same_value(self, column, value, negate=False):
             sel = np.logical_not(sel)
         return self.from_table_rows(self, sel)
 
+    def _filter_by_condition(self, f, conjunction, sel, data_filter):  # pragma: no cover
```

Comment:
Please stop abusing pragma: no cover.

This method should be tested, along with every other line of code in the table.py module. If codecov complains that the code you have moved around (but not changed) is not covered, make sure that the refactoring is done in a separate commit (so that the change can be inspected separately) and state that in the comment. Or write some tests. Basically any other way to approach it is better than marking actual code with "no cover".